libs/sdk-ui/src/base/results/dataAccessConfig.ts

```diff
@@ -8,22 +8,22 @@ import unescape from "lodash/unescape";
 const customEscape = (str: string) => str && escape(unescape(str));
 
 /**
- * @alpha
+ * @public
  */
 export type ValueFormatter = (value: DataValue, format: string) => string;
 
 /**
- * @alpha
+ * @public
  */
 export type HeaderTranslator = (value: string) => string;
 
 /**
- * Creates value formatter that uses @gooddata/numberjs to format raw measure values according
+ * Creates value formatter that uses `@gooddata/numberjs` to format raw measure values according
  * to the format string. By default, the format will strip away all the coloring information and
  * just return the value as string.
  *
- * @param separators - number separators to use. if not specified then numberjs defaults will be used
- * @alpha
+ * @param separators - number separators to use. if not specified then `numberjs` defaults will be used
```

Comment:
Nitpick: we should probably use the whole `@gooddata/numberjs` consistently (it is here and on another place in this file).

Reply:
fixed